code/datums/gamemodes/spy_theft.dm

```diff
@@ -126,6 +129,10 @@
 
 
 
+/datum/game_mode/spy_theft/New()
+	src.area_blacklist = list("AI Perimeter Defenses", "VR Test Area", "Storage Area")
```

Comment:
i feel like the storage area bug is better fixed by using abstract / concrete types

Reply:
The "Storage Area" turf is instantiated in the smuggler Z5 prefab, it's not abstract

Reply:
I'd rather change that tbh